submittal/src/main/java/com/kyriba/submittal/domain/Speaker.java

```diff
@@ -0,0 +1,21 @@
+package com.kyriba.submittal.domain;
+
+import java.util.List;
+
+
+/**
+ * @author M-ABL
+ */
+public class Speaker
+{
+  private long id;
+  private String email;

```

Comment:
Probably properties could be grouped in logical classes like Address, ContactData, etc.

Reply:
email here is more like a login

Reply:
ok, but comment still actual for other fields

Reply:
well, I have e.g. domain.dto.JobInfoDto. following your proposal I should also create domain.JobInfo domain object. What's the difference?